apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/KeysTests.cs

```diff
@@ -21,6 +21,14 @@ namespace Google.Cloud.Spanner.Data.Tests
 {
     public class KeysTests
     {
+        // Options will now be needed in all the tests.
```

Comment:
Given that these are reasonably short expressions anyway, are they actually worth having as fields? I suspect the tests might be clearer explicitly specifying `SpannerConversionOptions.Default` instead of `s_conversionOptions`. (Aside from anything else, it prompts the question "should we add a test for non-default options?")

Reply:
+1 to adding a test for non default options.